takoyaki/src/main/java/frc/robot/subsystems/Camera.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems;
+
+import java.util.List;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.util.Units;
+
+
+public class Camera {
+  private PhotonCamera camera;
+  private String name;
+  public Camera(String name) {
+    this.camera = new PhotonCamera(name);
+    this.name = name;
+  }
+
```

Comment:
nit: add `@Override` annotation to `periodic` method

Reply:
actually, this isn't even a subsystem. consider moving this class somewhere else, like put it into `frc.robot.utils` or something since it's a wrapper class. then, move this `periodic` method to your `CameraWithPID` class and then add the `@Override` there